### ISSUE TITLE
Why do I keep getting "Devices Error: 500 Internal Server Error"

### ISSUE THREAD

**Issue by batracy** [hjdhjd/homebridge-myq#915]
**Describe The Problem:**
I'm periodically getting an "Devices Error: 500 Internal Server Error"

**To Reproduce:**
Don't know. Just seems to happen.

**Logs:**
Most recent:
[11/16/2022, 8:54:30 AM] [myQ Garage Door] myQ API: https://devices.myq-cloud.com/api/v5.2/Accounts/5bef2969-1b14-41f8-8275-6881d77e4d15/Devices Error: 500 Internal Server Error
[11/16/2022, 8:54:30 AM] [myQ Garage Door] myQ API: Unable to update device status from the myQ API. Acquiring a new access token.

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Homebridge Version:** v1.5.1
* **Node Version:** v16.15.1
* **Homebridge-myQ Plugin Version**: v2.7.4
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** MacOS
* **Operating System and OS Version:** Monterey

**Comment by hjdhjd** [comment]
Expected behavior. The myQ API backend isn't the most reliable. Well documented in many other similar issues and the documentation itself.

TL;DR: ignore it. I just like being transparent about how reliable (or not) the myQ API can be at times.

Thanks for the report.
